{{cookiecutter.project_name}}/test/itk{{cookiecutter.filter_name}}Test.cxx

```diff
@@ -16,81 +16,82 @@
  *
  *=========================================================================*/
 
-#include "itk{{cookiecutter.filter_name}}.h"
+#include "itk{{ cookiecutter.filter_name }}.h"
```

Comment:
Will spaces here be a problem?

Reply:
The templating system (jinja2) supports the spaces, and they also help readability.